packages/shell-api/src/helpers.ts

```diff
@@ -937,9 +936,11 @@ export function processFLEOptions(
       );
     }
   } else {
+    // TODO: switch from any to KMSProviders (@mongosh/service-provider-core) once
+    // https://github.com/mongodb/node-mongodb-native/pull/4160/files is released.
     autoEncryption.kmsProviders = {
       ...fleOptions.kmsProviders,
-    } as KMSProviders;
+    } as any;
```

Comment:
```suggestion
    } as any as KMSProviders;
```

makes it a bit clearer what the intended type is?